tools/apiview/parsers/csharp-api-parser/CSharpAPIParserTests/TestData.cs

```diff
@@ -0,0 +1,255 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace CSharpAPIParserTests
+{
+    internal class TestData
+    {
+        public static string TokenJsonSchema = @"{
+    ""$schema"": ""https://json-schema.org/draft/2020-12/schema"",
```

Comment:
We might want to stay with `draft-07`
https://github.com/SchemaStore/schemastore/blob/master/CONTRIBUTING.md#best-practices

Reply:
I have used the schema that's generated from TSP json generator. We are not going to check in this schema to json schema store.